page_object/toDoListPageObject.ts

```diff
@@ -0,0 +1,48 @@
+import {expect} from '@playwright/test';
+
+const ulrTodolist = 'https://webdriveruniversity.com/To-Do-List/index.html'
+const newTodoPlaceholder = 'Add new todo';
+
+export class ToDoListPageObject{
+    async navigate(page: any){
+        await page.goto(ulrTodolist)
+    }
+
+    async checkPageTitle (page: any){
+        await expect(page).toHaveTitle('WebDriver | To Do List');
+   }
+
+    async fillAddTodoItem (page: any, value: string){
+        await page.getByPlaceholder(newTodoPlaceholder).fill(value)
+    }
+
+    async enterAddTodoItem (page: any){
+        await page.getByPlaceholder(newTodoPlaceholder).press('Enter')
+    }
+
+    async checkIfTaskWasAdded (page: any, value: string){
+        await expect(page.locator(`text=${value}`)).toHaveText(value)
+    }
+
+    async clickOnCompletedTask (page: any, value: string){
+        await page.locator(`text=${value}`).click()
+    }
+    async chceckIfTaskIsCompleted (page: any, value: string){
+        await expect(page.locator(`text=${value}`)).toHaveClass('completed')
+        await expect(page.locator(`text=${value}`)).not.toHaveClass('')
+    }
+
+    async checkHowManyElementsOnList (page: any, decimal: number){
+        await expect(page.locator('li')).toHaveCount(decimal);
```

Comment:
I know it is very simple page but please make Locator as variable in constructor.